docs/decisions/0009-usage-of-react-bootstrap.rst

```diff
@@ -0,0 +1,68 @@
+5. Usage of React-Bootstrap
+---------------------------
+
+
+Status
+------
+
+Adopted
+
+
+Context
+-------
+
+Paragon is built on top of the Bootstrap CSS Framework. The Paragon React component library aims to mirror Bootstrap's HTML and Javascript components. To date we have not expanded the Paragon React component library to include all Bootstrap components.
+
+ADR 5 Made the case for adoption of Reactstrap. ADR 8 abandoned the idea because adopting Reactstrap "comes at the cost of abstracting away raw html and introducing yet another API to learn and remember." Since then, we have added very few homegrown components. Designers and engineers are often needing components that exist in Bootstrap but not built as Paragon components.
+
+By not building these components ourselves and not adopting third-party react components, we are wasting time and generating confusion in the edX organization.
```

Comment:
> we are wasting time and generating confusion in the edX organization.

might recommend explaining why this is the case here with a concrete example.

Reply:
Added examples